### PR TITLE
Update url for "Transport for Germany" API

### PR DESCRIPTION
**Pull request by JulHelm**
Hi. Our german Bahn changet the URI to their APIs. Cheers. Julian

- [x] My submission is formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [x] My addition is ordered alphabetically
- [x] My submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column is padded with one space on either side
- [x] I have searched the repository for any relevant issues or pull requests
- [x] Any category I am creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

**Review by catcosmicice — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -918,7 +918,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Transport for Czech Republic](https://www.chaps.cz/eng/products/idos-internet) | Czech transport API | No | Yes | Unknown |
 | [Transport for Denver, US](http://www.rtd-denver.com/gtfs-developer-guide.shtml) | RTD | No | No | Unknown |
 | [Transport for Finland](https://digitransit.fi/en/developers/ ) | Finnish transport API | No | Yes | Unknown |
-| [Transport for Germany](http://data.deutschebahn.com/dataset/api-fahrplan) | Deutsche Bahn (DB) API | `apiKey` | No | Unknown |
+| [Transport for Germany](https://developer.deutschebahn.com/store/apis/list) | Deutsche Bahn (DB) API | `apiKey` | No | Unknown |
```
Comment (marekdano) [inline README.md:921]: `No` should be `Yes` since API is `https`. Can you please update it?

**Review by marekdano — CHANGES_REQUESTED** [review]
Thanks for the contribution @JulHelm!
My comment is below.

**Comment by pawelbr** [comment]
@JulHelm 
Can you commit the changes that @marekdano has suggested

**Comment by hjwk** [comment]
is the new address really correct ? It is impossible to access it, 
Should it perhaps be this one instead ? 
https://data.deutschebahn.com/dataset.groups.apis.html

Or perhaps simply keep the old one ? It is still accessible

**Comment by matheusfelipeog** [comment]
> is the new address really correct ? I know it might auth to access but the answer I get is weird :
> ...
> The old adress is still accessible

@hjwk You're right. This new link is broken, I will keep the current one.

Thanks for letting us know ;)

**Pull request closed without merging**